src/status_im/subs/wallet/wallet.cljs

```diff
@@ -522,16 +524,19 @@
  :<- [:profile/currency]
  :<- [:profile/currency-symbol]
  (fn [[aggregated-tokens color currency currency-symbol]]
-   (let [balance           (utils/calculate-balance-from-tokens {:currency currency
-                                                                 :tokens   aggregated-tokens})
-         formatted-balance (utils/prettify-balance currency-symbol balance)]
+   (let [balance             (utils/calculate-balance-from-tokens {:currency currency
+                                                                   :tokens   aggregated-tokens})
+         formatted-balance   (utils/prettify-balance currency-symbol balance)
+         token-values        (mapv #(utils/calculate-token-value {:token           %
+                                                                  :color           color
+                                                                  :currency        currency
+                                                                  :currency-symbol currency-symbol})
+                                   aggregated-tokens)
+         sorted-token-values (sort-by #(-> % :values :fiat-unformatted-value) > token-values)]
```

Comment:
again here we don't need `mapv`, let's prefer `map` unless we need it not to be lazy or we use the properties given by a vector (such as using `get` to extract items).

Reply:
That's right @ulisesmac, thanks!